msteams-platform/tabs/tabs-link-unfurling.md

```diff
@@ -250,6 +248,9 @@ Encoded URL:
 
 Whether you want to facilitate multitasking, enhance collaboration, or provide a focused user experience, Stageview has a mode to suit your requirements. Remember to consider the context in which your app is being used and ensure that your app works in that context.
 
+> [!NOTE]
+> The article is based on Teams JavaScript client library (TeamsJS) version 2.0.x. If you're using an earlier version, see [TeamsJS](how-to/using-teams-client-library.md) for guidance between the latest and earlier versions.
+
```

Comment:
@aaronmah, The developers have to know this information right in the start, hence moving the Note back to top.